config/routes.rb

```diff
@@ -4,18 +4,21 @@
   get 'about-us', to: 'home#about_us'
   get 'contacts', to: 'home#contacts'
 
-  get "/404", to: "errors#not_found"
-  get "/422", to: "errors#unacceptable"
-  get "/500", to: "errors#internal_error"
+  get '/404', to: 'errors#not_found'
+  get '/422', to: 'errors#unacceptable'
+  get '/500', to: 'errors#internal_error'
 
   devise_for :users, path: 'account', controllers: {
     registrations: 'users/registrations', sessions: 'users/track_sessions'
   }
 
   namespace :account do
+    concern :commentable do
```

Comment:
try something like [this](https://stackoverflow.com/questions/16376094/route-concern-and-polymorphic-model-how-to-share-controller-and-views/28466320#28466320)